### DISCUSSION TITLE
How to delete resource with force

### DISCUSSION THREAD

**Discussion by number21** [kubevela/kubevela#3092] [Q&A] [answered]
App delete stuck on this:
```
Message:               failed to sweep resourcetrackers to be deleted: failed to get resource networking.k8s.io/v1beta1/Ingress//default/helm-test-express-server: no matches for kind "Ingress" in version "networking.k8s.io/v1beta1"
    Reason:                ReconcileError
    Status:                False
    Type:                  Synced
```
Application is default that used to test kubevela deployment.

What was done: 
deployed application from this yaml - https://github.com/oam-dev/kubevela/blob/master/charts/vela-core/templates/test/test-application.yaml

after I figure out that i need use trait ingress-1-20 to match my k8s version

I just update trait in Application yaml and applied configuration to k8s.

Any ideas how to push application delete?

**Comment by wonderflow** [comment, marked as answer]
You can use `vela delete --force` in v1.3.x
